packages/trader/src/Modules/Trading/Components/Form/TradeParams/strike.jsx

```diff
@@ -54,10 +54,11 @@ const Strike = observer(() => {
                             i18n_default_text='If you buy a "<0>{{trade_type}}</0>" option, you receive a payout at expiry if the final price is {{payout_status}} the strike price. Otherwise, your “<0>{{trade_type}}</0>” option will expire worthless.'
                             components={[<strong key={0} />]}
                             values={{
-                                trade_type:
-                                    vanilla_trade_type === 'VANILLALONGCALL' ? localize('Call') : localize('Put'),
+                                trade_type: getContractSubtype(contract_type),
```

Comment:
same as https://github.com/binary-com/deriv-app/pull/9858/files#r1310994336

Reply:
But it should be non-localized, because we don't translate Call/ Put

Reply:
But I'll be safer to use localize in case if someone decide to translate, so I'll add back ternary